src/avt/Database/Ghost/avtStructuredDomainBoundaries.C

```diff
@@ -246,6 +251,8 @@ BoundaryHelperFunctions<T>::FillBoundaryData(int      d1,
             // find the match index ourselves
             mi = FindMatchIndex(d1,d2);
         }
+        if (mi < 0 || mi >= sdb->boundary[d2].neighbors.size())
+            EXCEPTION1(VisItException,"Bad Neighbor Index");
```

Comment:
Is there a way to provide more information that would be helpful to the user? Which domain contains the bad index, which index is bad? For here and elsewhere.

Reply:
Added information about which domain is missing the information. I did this here and everywhere a similar exception is thrown.